packages/handlebars/package.json

```diff
@@ -20,6 +20,9 @@
     "lodash": "^4.17.11",
     "promised-handlebars": "^2.0.1"
   },
+  "devDependencies": {
+    "@frctl/fractal": "^1.3.0"
+  },
   "peerDependencies": {
     "@frctl/fractal": "^1.1.0"
```

Comment:
This is probably not needed anymore then, right? 😊

Reply:
It is, since this tells the user that when installing `@frctl/handlebars`, core is also needed & the minimum required version is `1.1.0`.